src/Components/OATPropertyEditor/Internal/FormRootModelDetails/Internal/FormRootModelDetailsContent/PropertyDetailsEditorModalContent.tsx

```diff
@@ -526,6 +556,33 @@ const PropertyDetailsEditorModalContent: React.FC<IModalFormRootModelContentProp
                 />
             </div>
 
+            {/* version */}
+            {isDTDLModel(selectedItem) && (
+                <div className={propertyInspectorStyles.modalRow}>
+                    <Label className={classNames.label}>
+                        {t('OATPropertyEditor.version')}
+                    </Label>
+                    <Dropdown
+                        options={versionOptions}
+                        selectedKey={getDtdlVersion(selectedItem)}
+                        onChange={(
+                            _event: React.FormEvent<HTMLDivElement>,
+                            option?: IDropdownOption<string>
+                        ) => {
+                            debugger;
```

Comment:
Debugger should be removed and onChange should be moved to the commented out function on lines 137-140